edx_proctoring/backends/rest.py

```diff
@@ -313,7 +313,7 @@ def get_instructor_url(self, course_id, user, exam_id=None, attempt_id=None, sho
                 token['config'] = True
             if attempt_id:
                 token['attempt_id'] = attempt_id
-        encoded = jwt.encode(token, self.client_secret).decode('utf-8')
+        encoded = jwt.encode(token, self.client_secret)
```

Comment:
I have a confusion about this. Its a package and it will installed in `edx-platform` which has pyjwt==1.7.1.

Reply:
May be need to add check if it returns bytes decode it ?

Reply:
edx-proctoring and edx-platform will upgrade together.

Reply:
@awais786 To upgrade correctly, you need to first merge this PR and bump the library version to 4.0.0. Then go to the edx-platform pyjwt 2.1.0 upgrade PR, and also include in that PR the library upgrade of edx-proctoring to 4.0.0